jwst/datamodels/container.py

```diff
@@ -64,11 +69,12 @@ class ModelContainer(model_base.DataModel):
     # does not describe the data contents of the container.
     schema_url = "container.schema"
 
-    def __init__(self, init=None, **kwargs):
+    def __init__(self, init=None, allowed_exptypes=None, **kwargs):
 
-        super(ModelContainer, self).__init__(init=None, **kwargs)
+        super(ModelContainer, self).__init__(init=None, allowed_exptypes=None  , **kwargs)
```

Comment:
I assume pep8/flake8 won't like those whitespaces in "None  ,"

Reply:
Calls to `super()` can be simplified in python 3:
```
super().__init__(init=None, allowed_exptypes=None, **kwargs)
```

Reply:
Done